src/components/Resource.tsx

```diff
@@ -55,10 +55,11 @@ function useCalculatedViewWithState(props: InjectedPropTypes,
 export function LRC<P, R>(props: ResourcePropTypes<R> & P, _?: any): React.ReactElement | null {
     const context = useLinkRenderContext();
     const [error, setError] = React.useState<Error|undefined>(undefined);
+    const reset = React.useCallback(() => setError(undefined), []);
```

Comment:
the dependency here is `setError`

Reply:
setError will never change, because it comes from useState. Add it anyway?

Reply:
Is it defined as not changing? Otherwise it could change on every render

Reply:
Could not find a definition for it. I added the dependencies anyway.